master/buildbot/www/hooks/github.py

```diff
@@ -196,7 +191,10 @@ def _process_change(self, payload, user, repo, repo_url, project):
                 'branch': branch,
                 'revlink': commit['url'],
                 'repository': repo_url,
-                'project': project
+                'project': project,
+                'properties': {
+                                'github_distinct': commit.get('distinct', True)
```

Comment:
I dont think pep8 will be happy. we usually indent only once for this kind of code (instead of aligning with the {